config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
-end
+  namespace :api do
+    namespace :v1 do
+      resources :greetings
```

Comment:
- [ ] Since this is an API that has only a GET request, I recommend that you don't use `resources` and use only a get request, with APIs you should be careful of what to put, meaning you should only include what you're going to broadcast later!